tidysic/ordering.py

```diff
@@ -0,0 +1,24 @@
+from tidysic.tag import Tag
+from tidysic.formatted_string import FormattedString
+
+
+class OrderingStep:
+
+    def __init__(self, tag: Tag, formatted_string: FormattedString):
+        self.tag = tag
+        self.format = formatted_string
+
+
+class Ordering:
+
+    def __init__(self, steps: list[OrderingStep] = []):
+        self.steps = steps
+    
+    @property
```

Comment:
`is_terminal()` is not a property, just a class method.

Typing the return type as boolean could also be considered.